matlab_simulation/code v1.0/ME597-6-FastSLAM/newCode/callFS.m

```diff
@@ -0,0 +1,9 @@
+clear all; close all; clc;
+PoseInitial=[0,0,0]';
+dt=0.1;
+FinalTime=20;%final Time in sec
+MeasurementNoise=[0.001 0 0;0 0.001 0;0 0 0.0001];
+MotionNoise=[0.02 0;0 0.02];
+MapSize=20;
+ParticleSize=100;
+fastSLAM_func_p(PoseInitial,MotionNoise,MeasurementNoise,MapSize,FinalTime,dt,ParticleSize);
```

Comment:
While running this example the robot leaves the plot window for a portion of the simulation. The "fastSLAM_func_p" should likely be tweaked to plot a larger window into the world.